tests/test_ivs/test_ivs.py

```diff
@@ -0,0 +1,192 @@
+"""Unit tests for ivs-supported APIs."""
+import boto3
+from botocore.exceptions import ClientError
+from pytest import raises
+from moto import mock_ivs
+from re import fullmatch
+
+
+@mock_ivs
+def test_create_channel_with_name():
+    client = boto3.client("ivs")
```

Comment:
I should've spotted this earlier - but we have to specify a region for all tests, as there is not default region set in our CI.

Failing tests: https://github.com/getmoto/moto/actions/runs/6458787497/job/17534779886?pr=6894

Reply:
I understand. And I actually wondered why the tests generated by the `scaffold.py` script had a `region_name` in the calls, but the tests worked fine without that parameter. My environment doesn't have `AWS_DEFAULT_REGION` so that couldn't be it I thought. But I forgot about my `~/.aws/config` file. :facepalm: